### PR TITLE
Fix issue parsing Image Details from API due to incorrect struct json field

### PR DESCRIPTION
**Pull request by itsdax**
## Description

Issue is described here: https://github.com/cloudflare/cloudflare-go/issues/1378
API returns the metadata object under the field `meta`. However the SDK expects the field `metadata` thus it always drops this data from the response.

## Has your change been tested?

Yes - after changing the struct field for Metadata to `meta`, `GetImage` calls return images with Metadata field populated.

## Screenshots (if appropriate):

https://developers.cloudflare.com/api/operations/cloudflare-images-image-details

Refer to the response example of Image details. Specifically the `meta` field. The current SDK expects `metadata` and thus it always drops the metadata from the response.

## Types of changes

What sort of change does your code introduce/modify?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: images.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const (
 type Image struct {
 	ID                string                 `json:"id"`
 	Filename          string                 `json:"filename"`
-	Metadata          map[string]interface{} `json:"metadata,omitempty"`
+	Metadata          map[string]interface{} `json:"meta,omitempty"`
```
Comment (jacobbednarz) [inline images.go]: let's rename this field to `Meta` to match the API response

**Review by jacobbednarz — COMMENTED** [review]
thanks for the PR! added some notes inline.

File: images.go (hunk 1 above)
Comment (itsdax) [reply images.go]: Should I categorize this as a breaking change if this would break builds of applications referencing the old field?

Comment (jacobbednarz) [reply images.go]: yeah, that's fine to update the changelog entry.

**Pull request merged**